src/plugins/intel_cpu/src/extension.cpp

```diff
@@ -40,6 +42,10 @@ std::map<std::string, ngraph::OpSet> Extension::getOpSets() {
         NGRAPH_OP(LeakyReluNode, ov::intel_cpu)
         NGRAPH_OP(PowerStaticNode, ov::intel_cpu)
         NGRAPH_OP(SwishNode, ov::intel_cpu)
+        NGRAPH_OP(LoadConvertSaturation, ov::intel_cpu)
+        NGRAPH_OP(LoadConvertTruncation, ov::intel_cpu)
+        NGRAPH_OP(StoreConvertSaturation, ov::intel_cpu)
+        NGRAPH_OP(StoreConvertTruncation, ov::intel_cpu)
```

Comment:
To sum up, I have decided to have the separate ops because `StoreConvert` should have attribute `arithmetic_mode` that should be represenet as `Attribute`. I mean like `PadMode` for Pad op.
Also if we have separate `StoreConvertSaturation` and `StoreConvertTruncation` we should have separate `LoadConvertSaturation`/`LoadConvertTruncation` as well to have the same style :smile: